airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -32,7 +33,7 @@ class GoogleDisplayVideo360Hook(GoogleBaseHook):
 
     def __init__(
         self,
-        api_version: str = "v1",
+        api_version: str = "v2",
```

Comment:
We should consider this as a breaking change (even tough it has to be done because V1 is terminated by google)
You will need to edit provider.yaml and specify version 9.0.0 and also edit the changelog with relevant entry

Reply:
@eladkal I should change this or it should be updated by release coordinator?

Reply:
If the PR is not backward compatible then you need to make these changes (provider.yaml and changelog) as part of the PR.